cmake/projects/Qt/qt3d/hunter.cmake

```diff
@@ -88,5 +88,5 @@ hunter_pick_scheme(DEFAULT url_sha1_qt)
 hunter_download(
     PACKAGE_NAME Qt
     PACKAGE_COMPONENT "qt3d"
-    PACKAGE_INTERNAL_DEPS_ID "10"
```

Comment:
For good or for bad but Qt is not using `hunter_dump_cmake_flags` so PACKAGE_INTERNAL_DEPS_ID should not be changed.